File: python/fibonacci.py
# Fibonacci Series python language
import sys
 
if __name__ == "__main__":
   first = 0
   second = 1
   next = 0
 
   n = 30
 
   print("First ", n, " terms of Fibonacci series are :")
 
   c = 0
   while (c < n):
      if ( c <= 1 ):
         next = c
      else:
         next = first + second
         first = second
         second = next
      print(next)
      c += 1
 
   sys.exit(0)

File: go/fibonacci.go
// Fibonacci Series python language
package main

import ("fmt";"os")

func main() {
var first int = 0
var second int = 1
var next int = 0
var n int = 30
var c int = 0
 
   fmt.Printf("First %d terms of Fibonacci series are :", n)
 
   for (c < n) {
      if ( c <= 1 ) {
         next = c
      } else {
         next = first + second
         first = second
         second = next
      }
      fmt.Printf("%d\n", next)
      c += 1
   }
 
   os.Exit(0)
}
